dsc_lib/src/dscresources/command_resource.rs

```diff
@@ -3,11 +3,14 @@
 
 use jsonschema::JSONSchema;
 use serde_json::Value;
-use std::{collections::HashMap, env, io::{Read, Write}, process::{Command, Stdio}};
+use std::{collections::HashMap, env};
 use crate::{configure::{config_doc::ExecutionKind, {config_result::ResourceGetResult, parameters, Configurator}}, util::parse_input_to_json};
 use crate::dscerror::DscError;
 use super::{dscresource::get_diff, invoke_result::{ExportResult, GetResult, ResolveResult, SetResult, TestResult, ValidateResult, ResourceGetResponse, ResourceSetResponse, ResourceTestResponse, get_in_desired_state}, resource_manifest::{ArgKind, InputKind, Kind, ResourceManifest, ReturnKind, SchemaKind}};
 use tracing::{error, warn, info, debug, trace};
+use tokio::process::Command;
+use std::process::Stdio;
+use tokio::io::{BufReader, AsyncBufReadExt, AsyncWriteExt};
```

Comment:
combine the `tokio` to one line and keep fix the alpha ordering although I'd group the `crate` and `super` separately

Reply:
Updated.